app/scripts/lib/rpc-method-middleware/handlers/log-web3-usage.js

```diff
@@ -35,18 +35,19 @@ const recordedWeb3Usage = {}
  * @param {LogWeb3UsageOptions} options
  */
 function logWeb3UsageHandler(req, res, _next, end, { origin, sendMetrics }) {
-  const { action, name } = req.params[0]
+  const { action, path } = req.params[0]
 
   if (!recordedWeb3Usage[origin]) {
     recordedWeb3Usage[origin] = {}
   }
-  if (!recordedWeb3Usage[origin][name]) {
-    recordedWeb3Usage[origin][name] = true
+  if (!recordedWeb3Usage[origin][path]) {
+    recordedWeb3Usage[origin][path] = true
 
     sendMetrics({
-      event: `Website Accessed window.web3`,
+      matomo: true,
```

Comment:
Hey now, this isn't a real legacy event! We should try to avoid doing this, so we don't build up more work for ourselves in transitioning away from the legacy events.

Reply:
It was requested by Lorenzo to be tracked through our legacy analytics source

Reply:
Ah, I see. For what reason?

Reply:
The intent is to keep the production mixpanel instance as clean as possible while we're hammering out the final schema. If the event were to stick around long term we'd find the right user flow/analytics question to answer with this event -- but it likely won't persist beyond the deprecation.

Reply:
I think he just didn't want this event to pollute the prod dashboard, so to say.

Reply:
OK, makes sense.